docs/tutorials/send_receive.md

```diff
@@ -0,0 +1,154 @@
+description: "Steps to get a producer and a consummer exchanging messages over a deployed broker on OpenShift"
+draft: false
+images: []
+menu:
+  docs:
+    parent: "tutorials"
+weight: 110
+toc: true
+---
+
+### Prerequisite
+
+Before you start, you need to have access to a running Kubernetes cluster
+environment. A [Minikube](https://minikube.sigs.k8s.io/docs/start/) instance
+running on your laptop will do fine.
+
+### Step 1 - Deploy the operator
+
+#### create the namespace
+
+```shell script
+$ kubectl create namespace activemq-artemis-operator
+```
+
+Go to the root of the operator repo and install it:
+
+```shell script
+$ cd activemq-artemis-operator
+$ ./deploy/install_opr.sh
+```
+
+Deploy the Operator:
+
+```shell script
+$ kubectl create -f deploy/operator.yaml --namespace activemq-artemis-operator
+```
+
+Wait for the Operator to start (status: `running`).
+```shell script
+$ kubectl get pod --namespace activemq-artemis-operator
+```
+
+### Step 2 - Deploying the ActiveMQ Artemis Broker
+
+For this tutorial we need to:
+
+* have a broker that is able to listen to any network interface. For that we
+  setup an `acceptor` that will be listening on every interfaces on port
+  `62626`.
+* have queues to exchange messages on. These are configured by the broker
+  properties. Two queues are setup, one called `APP.JOBS` that is of type
+  `ANYCAST` and one called `APP.COMMANDS` that is of type `MULTICAST`.
+
+```shell script
+$ kubectl apply -f - <<EOF
+apiVersion: broker.amq.io/v1beta1
+kind: ActiveMQArtemis
+metadata:
+  name: default
+  namespace: activemq-artemis-operator
+spec:
+  acceptors:
+    - bindToAllInterfaces: true
+      name: acceptAll
+      port: 62626
+  brokerProperties:
+    - addressConfigurations."APP.JOBS".routingTypes=ANYCAST
+    - addressConfigurations."APP.JOBS".queueConfigs."APP.JOBS".routingType=ANYCAST
+    - addressConfigurations."APP.COMMANDS".routingTypes=MULTICAST
+  deploymentPlan:
+    image: 'quay.io/artemiscloud/activemq-artemis-broker-kubernetes:latest'
+    initImage: 'quay.io/artemiscloud/activemq-artemis-broker-init:latest'
+    requireLogin: false
+    size: 1
+EOF
+activemqartemis.broker.amq.io/default created
+```
+
+Wait for the Broker to start (status: `running`).
+
+```shell script
+$ kubectl get pod --namespace activemq-artemis-operator
+NAME                                                   READY   STATUS    RESTARTS
+activemq-artemis-controller-manager-66dbbbb679-hfmsn   1/1     Running   5 (3d22h ago)
+default-ss-0                                           1/1     Running   0
+```
+
+### Step 3 - Forwarding ports
+
+```shell script
+$ kubectl port-forward default-ss-0 62626 -n activemq-artemis-operator
+Forwarding from 127.0.0.1:62626 -> 62626
+Forwarding from [::1]:62626 -> 62626
+```
+
+### Step 4 - Exchanging messages between a producer and a consumer
+
+#### ANYCAST
+
+```shell script
+$ ./artemis producer --destination APP.JOBS  --url tcp://localhost:62626
```

Comment:
The users could not know where to find the artemis script, the doc could include a pointer to the Apache ActiveMQ Artemis download/install page